cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/file/FileSetDataset.java

```diff
@@ -60,11 +61,12 @@
  */
 public final class FileSetDataset implements FileSet, DatasetOutputCommitter {
 
-  private static final Logger LOG = LoggerFactory.getLogger(FileSetDataset.class);
-
   public static final String FILESET_VERSION_PROPERTY = "fileset.version";
   static final String FILESET_VERSION = "2";
 
+  private static final Logger LOG = LoggerFactory.getLogger(FileSetDataset.class);
+  private static final String HADOOP_UMASK_PROPERTY = FsPermission.UMASK_LABEL; // fs.permissions.umask-mode
```

Comment:
Note: this seems like a duplication (I also defined it in MapReduceRuntimeService) but I did not want these classes to depend on each other. It is only a repetition of a Hadoop constant anyway.